apt/minimization/minimizer.py

```diff
@@ -43,7 +47,13 @@ class GeneralizeToRepresentative(BaseEstimator, MetaEstimatorMixin, TransformerM
 
     features : list of str, optional
         The feature names, in the order that they appear in the data.
-
+    categorical_features: list of str, optional
+        The list of categorical features should only be supplied when
+         passing data as a pandas dataframe.
+    quasi_identifiers: The indexes of the features that need to be
```

Comment:
quasi-identifiers is actually a very specific term in the context of anonymization/re-identification. I suggest here to call them 'target_features' or 'features_to_minimize', and also remove what is written in the parentheses.